src/Components/AddItem.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class AddItem extends React.Component {
+  render() {
+    return <h1>Add Item</h1>;
+  }
+}
+
+export default AddItem;
```

Comment:
I am just curious why you have chosen to use class component instead of functional components. Are there benefits to using class components as opposed to functional components in this situation/problem you are trying to solve? There seems to be [a lot of benefits of using functional components as opposed to class components](https://programmingwithmosh.com/react/react-functional-components/).